### ISSUE TITLE
Nuxt - Having issue with components state not showing with vuedevtools browser extension v7

### ISSUE THREAD

**Issue by bsiguret** [vuejs/devtools#659]
Hello! Everything is in the title:

My Vue Devtool extension updated itself to v7, and I have trouble with most of my components having no state showing in the browser extension.

I went back to the v6 version to fix it.

The project is on Nuxt v3.13.2 (associated with vue v3.5.5)

Here are screenshots of what I'm saying:

v6
<img width="554" alt="image" src="https://github.com/user-attachments/assets/8986a617-681d-4d34-bc70-10f8506c335f">
v7
<img width="540" alt="image" src="https://github.com/user-attachments/assets/1de1bda7-44ad-4204-a72a-6d01a704aebc">

I investigated a bit and the components that aren't working are inside a v-for, but it's not a 100% fail rate and maybe not true.

I can't share the project since it's a private project

**Comment by wynged** [comment]
Same issue here.  Here's my empty state, showing some console logs, as well as the location in the code of the log issue, not sure how helpful it is but 🤷 
![image](https://github.com/user-attachments/assets/1c5c43b6-251e-4670-81f4-9db1b3b13b93)
![image](https://github.com/user-attachments/assets/6817f3a5-653a-456a-8824-b542a7be6310)

**Comment by webfansplz** [comment]
Try to fix it in v7.6.1 (pending review), Please help me confirm whether it is fixed when it passes the audit on chrome store, thank you

**Comment by bsiguret** [comment]
@webfansplz Not fixed for me in v7.6.1, the same problem. 

<img width="476" alt="image" src="https://github.com/user-attachments/assets/2a1069c5-83ab-4ee6-87d4-329a25205f41">
<img width="789" alt="image" src="https://github.com/user-attachments/assets/a9284d3c-d1ca-4fe6-844b-e4a53d07332a">


I don't see your PR changes in the source code (the optional chaining) and I don't think this is normal. I tried to remove and reinstall the extension + restart my browser without change. Should I do something else?

Thank you 👍

**Comment by webfansplz** [comment]
> @webfansplz Not fixed for me in v7.6.1, the same problem.
> 
> <img alt="image" width="476" src="https://private-user-images.githubusercontent.com/30038522/382028882-2a1069c5-83ab-4ee6-87d4-329a25205f41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTI2MzMsIm5iZiI6MTczMDM5MjMzMywicGF0aCI6Ii8zMDAzODUyMi8zODIwMjg4ODItMmExMDY5YzUtODNhYi00ZWU2LTg3ZDQtMzI5YTI1MjA1ZjQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2MzIxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1MzM4OTE1YTlkNjdmOGRkOWYxOWZhM2U0NzRkOTE0MTg4M2QzMDMwMWViOTJkMTA0ZDZiZmIxOWM2YTMzNTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.v0Bv9A4Qs5T6j_n-ZfHM2DBme7sP96lKEmObR2ivQCw"> <img alt="image" width="789" src="https://private-user-images.githubusercontent.com/30038522/382029492-a9284d3c-d1ca-4fe6-844b-e4a53d07332a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTI2MzMsIm5iZiI6MTczMDM5MjMzMywicGF0aCI6Ii8zMDAzODUyMi8zODIwMjk0OTItYTkyODRkM2MtZDFjYS00ZmU2LTg0NGItZTRhNTNkMDczMzJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2MzIxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwYTE2ODQ0OTFiZjc4ZWYwYjQxM2VkMWYyYWM0NThmOGJhYjFhYTk1NTFmZTNhZTAwOWFhNWMwZDNmNDU1YjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Fy_62yrXmeMVlNwpS1o6PwkrD-MufsotLFQoGFsOsh0">
> I don't see your PR changes in the source code (the optional chaining) and I don't think this is normal. I tried to remove and reinstall the extension + restart my browser without change. Should I do something else?
> 
> Thank you 👍
[devtools-chrome.zip](https://github.com/user-attachments/files/17590529/devtools-chrome.zip)
Can you try this zip?

**Comment by webfansplz** [comment]
> > @webfansplz Not fixed for me in v7.6.1, the same problem.
> > <img alt="image" width="476" src="https://private-user-images.githubusercontent.com/30038522/382028882-2a1069c5-83ab-4ee6-87d4-329a25205f41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTI2MzMsIm5iZiI6MTczMDM5MjMzMywicGF0aCI6Ii8zMDAzODUyMi8zODIwMjg4ODItMmExMDY5YzUtODNhYi00ZWU2LTg3ZDQtMzI5YTI1MjA1ZjQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2MzIxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1MzM4OTE1YTlkNjdmOGRkOWYxOWZhM2U0NzRkOTE0MTg4M2QzMDMwMWViOTJkMTA0ZDZiZmIxOWM2YTMzNTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.v0Bv9A4Qs5T6j_n-ZfHM2DBme7sP96lKEmObR2ivQCw"> <img alt="image" width="789" src="https://private-user-images.githubusercontent.com/30038522/382029492-a9284d3c-d1ca-4fe6-844b-e4a53d07332a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzOTI2MzMsIm5iZiI6MTczMDM5MjMzMywicGF0aCI6Ii8zMDAzODUyMi8zODIwMjk0OTItYTkyODRkM2MtZDFjYS00ZmU2LTg0NGItZTRhNTNkMDczMzJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE2MzIxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwYTE2ODQ0OTFiZjc4ZWYwYjQxM2VkMWYyYWM0NThmOGJhYjFhYTk1NTFmZTNhZTAwOWFhNWMwZDNmNDU1YjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Fy_62yrXmeMVlNwpS1o6PwkrD-MufsotLFQoGFsOsh0">
> > I don't see your PR changes in the source code (the optional chaining) and I don't think this is normal. I tried to remove and reinstall the extension + restart my browser without change. Should I do something else?
> > Thank you 👍

[devtools-chrome.zip](https://github.com/user-attachments/files/17590529/devtools-chrome.zip)
@bsiguret Can you try this zip?

**Comment by webfansplz** [comment]
@bsiguret I have just released v7.6.2 and await review. You can also try again after passing the review (if there is any latest news, please let me know, thank you).

**Comment by bsiguret** [comment]
> @bsiguret Can you try this zip?

No change with the zip sorry 😢 I'll test the 7.6.2 if you changed something. Have a nice evening 😄

**Comment by webfansplz** [comment]
> > @bsiguret Can you try this zip?
> 
> @webfansplz No change with the zip sorry 😢 I'll test the 7.6.2 if you changed something. Have a nice evening 😄

This is a bit strange, can you again make sure to uninstall all Vue devtools in case and restart the browser to try to install it? (I reproduce the issue yesterday and confirmed that I had fixed it

**Comment by webfansplz** [comment]
@wynged How about you? It’s working fine for you?

**Comment by bsiguret** [comment]
@webfansplz maybe it will be fixed later. When I check the source file of the error, I see that it comes from a node_module @vue -> devtools-core. I don't know how this package works with the extension so maybe I'll need to wait for a Vue and then a Nuxt version update.

<img width="572" alt="image" src="https://github.com/user-attachments/assets/43fcb6c4-66bb-43cb-83d8-1d01adce3f83">

I'm sorry if I'm making a dumb error. Anyway, it's not blocking my development since I can work with the previous version just fine. I'll wait a couple of weeks and ping you again on this issue if necessary.

Thank you for your time 🙏

**Comment by wynged** [comment]
it's working better..... I saw some state, but when I popped up a modal in my app and tried to select the modal to view it's state the state wouldn't update, although I was scrolled to the component in the tree on the left. When I closed the modal I got the state to swap when I selected other components, but I also saw this error.

It did seem to get better when I tried again, but when I first popup our modal it doesn't seem to be something that the component picker can see, it doesn't get highlighted at all.
![image](https://github.com/user-attachments/assets/946940f3-66a5-4de5-a25a-119896054ed5)
